datamover/pkg/drivers/https/migration.go

```diff
@@ -615,7 +266,9 @@ func runjob(in *pb.RunJobRequest) error {
 		}
 
 		//Do migration for each object.
-		go doMove(ctx, objs, capa, th, srcLoca, destLoca, in.RemainSource, &j)
+		//go doMove(ctx, objs, capa, th, srcLoca, destLoca, in.RemainSource, &j)
```

Comment:
please remove this commented old code. Thanks

Reply:
Done, thanks !